### PR TITLE
Extend product webhook with attributes

### PR DESCRIPTION
**Pull request by krzysztofwolski**
I want to merge this change because product and variant attributes were missing in webhook data.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [x] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [x] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by IKarbowiak — COMMENTED** [review]
Some tests should be added.

File: saleor/webhook/payloads.py [hunk 1]
```diff
@@ -214,26 +239,21 @@ def generate_product_payload(product: "Product"):
...
-            "variants": (
-                lambda p: p.variants.annotate_quantities().all(),
-                product_variant_fields,
-            ),
+        },
+        extra_dict_data={
+            "attributes": serialize_attribute_assignments(product),
+            "variants": [
+                json.loads(generate_variant_payload(v))[0]
+                for v in product.variants.annotate_quantities().all()
```
Comment (korycins) [inline saleor/webhook/payloads.py:255]: IMO, the prefetch_related whould be usefull here

File: saleor/webhook/serializers.py [hunk 2]
```diff
@@ -23,3 +24,23 @@ def serialize_checkout_lines(checkout: "Checkout") -> List[dict]:
             }
         )
     return data
+
+
+def serialize_attribute_assignments(
+    object_with_attributes: Union["Product", "ProductVariant"]
+):
+    data = []
+    for assignment in object_with_attributes.attributes.all():
```
Comment (korycins) [inline saleor/webhook/serializers.py:33]: Don't we need prefetches here?

File: templates/templated_email/source/shared/generic_information.mjml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<mj-section background-color="#ffffff" padding-left="10px" padding-right="10px" padding-top="0px">
```
Comment (korycins) [inline templates/templated_email/source/shared/generic_information.mjml:1]: @krzysztofwolski  Are you sure that this should be included?

**Comment by krzysztofwolski** [comment]
We changed approach in algolia integration - instead of extending hook payload we pull missing data from gql api. This way we will be able to get channel pricing we require.

**Pull request closed without merging**